### DISCUSSION TITLE
Approve subset of PR changes

### DISCUSSION THREAD

**Discussion by jasonvarga** [community/community#7484] [Pull Requests]
It would be nice to be able to approve a subset of changes on a PR.

What I mean can be better explained with a workflow:

- Contributor opens a PR.
- Reviewer sees something immediately obvious and requests a change.
- Contributor pushes those changes.
- Reviewer wants to mark **just the requested changes** as approved, but not the PR as a whole.

Perhaps an option to change the state back to just "awaiting review".

Thanks!

**Comment by 94rain** [comment]
I think this can be done by "marking the conversation as resolved"?
![image](https://user-images.githubusercontent.com/21193371/140829542-e394fee6-9aa0-4380-b7b1-db808d0e9e05.png)

**Reply by jasonvarga** [reply]
It looks like even if the conversations are resolved, it still says changes were requested.
